src/components/screens/monitor/delegates/forgingDetails.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { Link } from 'react-router-dom';
+import Box from '../../../toolbox/box';
+import BoxHeader from '../../../toolbox/box/header';
+import BoxContent from '../../../toolbox/box/content';
+import styles from './forgingDetails.css';
+import LiskAmount from '../../../shared/liskAmount';
+import { tokenMap, initialLSKSupply } from '../../../../constants/tokens';
+import routes from '../../../../constants/routes';
+
+const ForgingDetails = ({
+  t, delegates, sortDirection, networkStatus,
+}) => {
+  const delegatesList = sortDirection && sortDirection.includes('asc') ? [...delegates.data] : [...delegates.data].reverse();
+  const totalForged = networkStatus && networkStatus.data.supply - initialLSKSupply;
+
+  const lastForger = [...delegatesList].sort(
+    (a, b) =>
+      b.lastBlock && a.lastBlock && b.lastBlock.height - a.lastBlock.height,
+  )[0];
+
+  const nextForgers = [...delegatesList]
+    .filter(d => d.forgingTime)
+    .slice(0, 10)
+    .sort((a, b) => a.forgingTime - b.forgingTime);
```

Comment:
🐛 It's not getting the correct delegates. It should first `sort` then `slice`.

Reply:
✅